CHANGELOG.md

```diff
@@ -7,14 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0](https://github.com/jeertmans/languagetool-rust/compare/v2.1.4...v2.1.5)
 
-- Added support for Mardown and Typst files.
+- Added support for Markdown and Typst files.
```

Comment:
Either this is work for another PR, and we should remove this line, or did you plan to actually implement it? Because I think this was still W.I.P. from my side.

Reply:
I see, I can remove it for now then, though there is a mention of the type in the arguments for the CLI too